classes/binconfig.oeclass

```diff
@@ -1,13 +1,34 @@
 ## Class for packages having binconfig files.
 ##
 ## Rewrites the paths in the binconfig files so that it contains valid dirs.
+##
+## @var BINCONFIG_FILES   Space separated list of glob patterns to match files
+##                        for ${PN}-dev (defaults to "${bindir}/*-config")
+## @var BINCONFIG_GLOB    Glob pattern to match files that need fixup (defaults
+##                        to "${BINCONFIG_FILES}")
+## @var BINCONFIG_PACKAGE Name of cross package with symlink to binconfig files
+##                        (defaults to "${PN}-binconfig")
+## @var BINCONFIG_LIB     Name of package that depends on binconfig package
+##                        (default to "${PN}")
+## @var BINCONFIG_FIXUP_STRIP_DIRS
+##                        Space separated list of directories to fixup in binconfig
+##                        files (default to "${HOST_SYSROOT} ${TARGET_SYSROOT} ${D}
+##                        ${B} ${S}")
 
 require conf/meta.conf
 
 BINCONFIG_FILES ?= "${bindir}/*-config"
 BINCONFIG_GLOB ?= "${BINCONFIG_FILES}"
 FILES_${PN}-dev += "${BINCONFIG_FILES}"
 BINCONFIG_FIXUP_STRIP_DIRS ?= "${HOST_SYSROOT} ${TARGET_SYSROOT} ${D} ${B} ${S}"
+BINCONFIG_PACKAGE ?= "${PN}-binconfig"
+BINCONFIG_LIB ?= "${PN}"
```

Comment:
More naming.  It seems weird to call it BINCONFIG_LIB when it is just the name of a package containing one (or more) binconfig scripts.

Maybe BINCONFIG_CROSS_PACKAGE and BINCONFIG_PACKAGE instead of BINCONFIG_PACKAGE and BINCONFIG_LIB.